assets/oag.css

```diff
@@ -230,6 +238,54 @@
 		border: 1px solid transparent;
 		background-color: #F4F4F2;
 	}
+
+	.wb-tabs.vertical {
+		/* Vertical tab styling */
+		display: flex;
+		height: 100%;
+		[role="tablist"] {
+			width: max-content;
```

Comment:
I'm not sure I like the fact that the width of the tabs changes depending on which one is selected.